torchaudio/datasets/speechcommands.py

```diff
@@ -79,9 +90,25 @@ def __init__(self,
                     download_url(url, root, hash_value=checksum, hash_type="md5")
                 extract_archive(archive, self._path)
 
-        walker = walk_files(self._path, suffix=".wav", prefix=True)
-        walker = filter(lambda w: HASH_DIVIDER in w and EXCEPT_FOLDER not in w, walker)
-        self._walker = list(walker)
+        def _load_list(*filenames):
+            output = []
+            for filename in filenames:
+                filepath = os.path.join(self._path, filename)
+                with open(filepath) as fileobj:
+                    output += [os.path.join(self._path, line.strip()) for line in fileobj]
```

Comment:
I'm surprised our Windows CI job does not raise FileNotFound error with this implementation. If my understanding is correct, as previously noted, `os.path.join(self._path, line.strip())` would produce strings that look like `speech_commands\speech_commands_v0.01/testing_list.txt`, which is an invalid path on Windows.

Reply:
Yeah, that's what I see too. It seems like `os.path.normpath` would be useful here.